nav2_lifecycle_manager/src/lifecycle_manager.cpp

```diff
@@ -78,19 +75,23 @@ LifecycleManager::LifecycleManager()
   transition_label_map_[Transition::TRANSITION_DEACTIVATE] = std::string("Deactivating ");
   transition_label_map_[Transition::TRANSITION_UNCONFIGURED_SHUTDOWN] =
     std::string("Shutting down ");
+}
 
+LifecycleManager::~LifecycleManager()
+{
+  RCLCPP_INFO(get_logger(), "Destroying %s", get_name());
```

Comment:
```suggestion
  RCLCPP_INFO(get_logger(), "Destroying %s", get_name().c_str());
```

Reply:
get_name() is already a char* not a std::string. There is no .c_str() method.